src/types.ts

```diff
@@ -85,6 +85,7 @@ export type DataLinkConfig = {
   field: string;
   url: string;
   datasourceUid?: string;
+  urlDisplayLabel?: string;
```

Comment:
We can remove this. It's not available as an option in the config editor and the current frontend implementation uses an empty string for the data link `title` anyway.